rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/executor/BakePoller.groovy

```diff
@@ -144,18 +154,43 @@ class BakePoller implements ApplicationListener<ContextRefreshedEvent> {
 
   void updateBakeStatusAndLogs(String statusId) {
     BakeStatus bakeStatus = executor.updateJob(statusId)
+    Id bakesId = registry.createId('bakes')
 
     if (bakeStatus) {
       if (bakeStatus.state == BakeStatus.State.COMPLETED) {
         completeBake(statusId, bakeStatus.logsContent)
-      }
 
-      bakeStore.updateBakeStatus(bakeStatus)
+        bakeStore.updateBakeStatus(bakeStatus)
+
```

Comment:
I can probably only make the call if the id is actually set. Ok, let me try to rework it a bit.